iOS_Labs/Views/Photos/SubViews/Img.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Image.swift
+//  iOS_Labs
+//
+//  Created by Daniil Oleshchuk on 02.12.2021.
+//
+
+import SwiftUI
+
+
+struct Img: View {
```

Comment:
We can call it `ImageView` because `Img` is not very good name for UI element. It would suit for some temporary variable but not thins, I think.